portal-impl/src/com/liferay/portlet/documentlibrary/service/impl/DLFileEntryLocalServiceImpl.java

```diff
@@ -2771,6 +2781,81 @@ private long _getValidFileEntryTypeId(
 		}
 	}
 
+	private void _initializeFileEntryPermissionForDLFolder(
```

Comment:
I think the function name is `initializeInheritedPermissionFromParentFolder` would be better. What do you think?